src/components/Grade/grade.module.scss

```diff
@@ -1,3 +1,25 @@
+.A { 
+    background-color: var(--seed-color-primary);
+}
+
+.B { 
+    background-color: #da5156;
+}
+.C {
+    background-color: #9e3c40;
+}
+
+.D, .F { 
+    background-color: #612527;
+}
+.A, .B, .C, .D, .F { 
+    width: 25px;
+    text-align: center;
+    border-radius: 50%;
+    padding: 10px;  
+    margin: 8px;
+    color: #ffeaea;
+}
 .offset {
```

Comment:
Let's let Paul come up with some mappings for the colors. For now can you just use the component I built? I know all the colors will be red but it'll fix itself when we update that one component. No need for all these extra styles